crypto/x509/x509_vpm.c

```diff
@@ -507,6 +515,7 @@ static const X509_VERIFY_PARAM default_table[] = {
      0,                         /* check time to use */
      0,                         /* inheritance flags */
      X509_V_FLAG_TRUSTED_FIRST, /* flags */
+     0,                         /* eku */
```

Comment:
```suggestion
     NID_undef,                         /* eku */
```
And similar cases below.